modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala

```diff
@@ -84,8 +84,7 @@ final class UpdateService[F[_]](
             util.divideOnError(prj)(sbtAlg.getUpdatesForProject)(_.halve.toList.flatMap {
               case (p1, p2) => List(p1, p2)
             }) { (failedP: ArtificialProject, t: Throwable) =>
-              println(s"failed finding updates for $failedP")
-              println(t)
+              logger.error(t)(s"failed finding updates for $failedP error=${t.getMessage}")
```

Comment:
`logger.error(...)` is a pure expression and needs to be composed via `>>` with the following expression to have an effect.

Reply:
`logger.error(...).as(List.empty[Update.Single])` is even simpler since the result is just a pure.

Reply:
:+1: This is a nice application of the monad law: `flatMap(_ => pure(x)) <-> map(_ => x)`.